### ISSUE TITLE
[Bug Report] transfer doesn't work when v-model is empty

### ISSUE THREAD

**Issue by houxg** [element-plus/element-plus#1152]
### Element Plus version
1.0.1-beta.17

### OS/Browsers version
Version 87.0.4280.88 (Official Build) (x86_64)

### Vue version
3.0.4

### Reproduction Link
https://codepen.io/pen/?&prefill_data_id=eedd6401-a759-4802-9d91-efdf50a29451

### Steps to reproduce
1. open reproduction link
2. select items in left side
3. click arrow button

### What is Expected?
selected items should show in right side

### What is actually happening?
selected items not show

**Comment by zazzaz** [comment]
please provide a valid reproduction Link

**Comment by houxg** [comment]
@zazzaz actually it's the example code in [element-plus.org](https://element-plus.org/#/zh-CN/component/transfer#ke-sou-suo)，click “在线运行” button and run the example code in codepen.io, you will find something.

**Comment by zazzaz** [comment]
@houxg 

This might be a bug in vue@3.0.4, you can use 3.0.3 at the moment.
```
<script src="https://unpkg.com/vue@3.0.3/dist/vue.global.js"></script>
```

**Comment by zazzaz** [comment]
this should have been fixed in `vue@3.0.5` https://github.com/vuejs/vue-next/commit/c59897c7b0dbd82b5bbf3fbca945c0639ac37fb8
